Source/WebCore/css/CSSProperties.json

```diff
@@ -7414,8 +7408,8 @@
         },
         "-webkit-backdrop-filter": {
             "codegen-properties": {
-                "related-property": "backdrop-filter",
-                "conditional-converter": "FilterOperations",
+                "cascade-alias": "backdrop-filter",
+                "skip-builder": true,
```

Comment:
Would this break `backdrop-filter` on down levels where `cssUnprefixedBackdropFilterEnabled` is turned off?

Reply:
You can provide an array to `codegen-properties` fwiw, with the first item in the list being for `ENABLE_UNPREFIXED_BACKDROP_FILTER` and the other one being for `!ENABLE_UNPREFIXED_BACKDROP_FILTER`. 

See `font-stretch` for instance

Reply:
> Would this break `-webkit-backdrop-filter` on down levels where `cssUnprefixedBackdropFilterEnabled` is turned off?

I don't think so? This only affects what happens during cascading.

Reply:
I guess it would be weird to alias to a property that's disabled, but I trust your judgment on whether that's fine to do.